ui/src/app/views/workspace/component.ts

```diff
@@ -330,15 +332,40 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
             this.resetAlerts();
           }, 5000);
         }, () => {
-          this.alertCategory = 'alert-danger';
-          this.alertMsg = 'There was an issue while saving file(s) please try again later';
-          this.showAlerts = true;
-          setTimeout(() => {
-            this.resetAlerts();
-          }, 5000);
+          this.handleLocalizeError();
         });
   }
 
+  localizeAllFiles(): void {
+    this.errorHandlingService.retryApi(this.workspacesService
+        .getBucketFilesList(this.wsNamespace, this.wsId, 'createCluster'))
+        .subscribe(
+            fileList => {
+              this.fileList = this.notebookList;
```

Comment:
The client shouldn't be telling the server what files to localize... the server should just fetch the list of files from GCS and localize them directly. So the endpoint for localizing notebooks should just be "localize workspace files" and do this logic internally.

Does that make sense?

Reply:
Fixed